libvirt.go

```diff
@@ -211,7 +226,7 @@ func (d *Driver) validateNetwork() error {
 }
 
 func (d *Driver) PreCreateCheck() error {
-	conn, err := d.getConn()
+	conn, err := d.getSystemConn()
```

Comment:
getting the  libvirt version shoulod be fine with getSessionConn?

Reply:
Nope, the network lives on the system connection and except for using it in a session connection VM (and even that as a bridge), we can't access the `crc` network.

Reply:
Ah! I missed the fact that `validateNetwork` gets its own connection so yes, we don't need to use system connection here.